rsconnect/api.py

```diff
@@ -661,7 +667,7 @@ def validate_rstudio_server(
                 raise RSConnectException("Failed to verify with {} ({}).".format(server.remote_name, exc))
 
     @cls_logged("Making bundle ...")
-    def make_bundle(self, func: Callable, *args, **kwargs):
+    def make_bundle(self, func: Callable[..., object], *args: object, **kwargs: object):
```

Comment:
I added the `object` type to a number of annotations, so that the places which call this function won't be flagged.

I used `object` because it makes the fewest assumptions about the object, but it may make sense to refine these these types in the future.